packages/richtext-lexical/src/index.ts

```diff
@@ -89,6 +90,64 @@ export function lexicalEditor(props?: LexicalEditorProps): LexicalRichTextAdapte
       })
     },
     editorConfig: finalSanitizedEditorConfig,
+    outputSchema: ({ isRequired }) => {
```

Comment:
How does this type work with custom lexical extensions? They will have to override the outputSchema to add in their own extra poperties?

Reply:
It does not allow custom features to add their own types yet, see this comment: https://github.com/payloadcms/payload/pull/4036/files#diff-012c3c92a58effd50cd58aa6a49d345fef3bde25eff1a5d3a16d8b90f21d170cR98

In fact, no lexical node is typed fully here. It only types the root node, as well as the properties which we know are always there.